tool/tctl/common/resource_command.go

```diff
@@ -808,16 +871,28 @@ func (rc *ResourceCommand) createLoginRule(ctx context.Context, client auth.Clie
 	}
 
 	loginRuleClient := client.LoginRuleClient()
+	_, err = client.LoginRuleClient().GetLoginRule(ctx, &loginrulepb.GetLoginRuleRequest{Name: rule.Metadata.Name})
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	exists := (err == nil)
 	if rc.IsForced() {
 		_, err := loginRuleClient.UpsertLoginRule(ctx, &loginrulepb.UpsertLoginRuleRequest{
 			LoginRule: rule,
 		})
 		return trail.FromGRPC(err)
 	}
+	if exists && !rc.force{
+		return trace.AlreadyExists("login rule '%s' already exists, use -f flag to override", rule.Metadata.GetName())
```

Comment:
```suggestion
		return trace.AlreadyExists("login rule '%s' already exists", rule.Metadata.GetName())
```

telling the user to use `-f` to override in these errors is redundant as the calling code already does that

https://github.com/hegurhurgurk/teleport/blob/c22559cb8839016b87c4b9c7a26a078288178cd2/tool/tctl/common/resource_command.go#L302-L307

(I know you're just copy/pasting what was already there, but let's fix it now). Let's also go through and make all of these `AlreadyExists` errors consistent for all the `CreateHandlers`: start lowercased (except for acronyms), no period, no mention of using -f.